nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -187,6 +187,10 @@ nifi.security.user.login.identity.provider=${nifi.security.user.login.identity.p
 nifi.security.ocsp.responder.url=${nifi.security.ocsp.responder.url}
 nifi.security.ocsp.responder.certificate=${nifi.security.ocsp.responder.certificate}
 
+# Custom TLS Cipher Suites (Override JRE Defaults)
+# nifi.web.https.includeciphersuites=
```

Comment:
Rather than have commented out properties, can you include them with just empty values?

Reply:
Definitely can be empty values.